src/metabase/models/card.clj

```diff
@@ -211,7 +211,7 @@
 
 (defn- create-actions-when-is-writable! [{is-write? :is_write card-id :id}]
   (when is-write?
-    (let [{action-id :id} (db/insert! action/Action {:type "query"})]
+    (let [{action-id :id} (db/insert! action/Action {:type "row"})]
```

Comment:
The DesignDoc says we have `row` and `bulk` type actions - and we're hardcoding to `row` for now. The presence of the `query_action` row makes the action execute a query.